<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GearsDialog</class>
 <widget class="QDialog" name="GearsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nouvel équipement</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nouvel équipement</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">color:dodgerblue; 
font-size:16px;
 background-color:lightgray;
font-weight:600;</string>
           </property>
           <property name="text">
            <string>Type</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbType"/>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true">color:dodgerblue; 
font-size:16px;
 background-color:lightgray;
font-weight:600;</string>
           </property>
           <property name="text">
            <string>Marque</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbMarque">
           <item>
            <property name="text">
             <string>B'Twin</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Modèle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leModele"/>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Date d'achat</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="deAchat">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Date de 1ère utilisation</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="deUtilisation">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>KM initial</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leKMInitial">
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly|Qt::ImhFormattedNumbersOnly</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>KM cumulés</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leKMCumules">
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly|Qt::ImhFormattedNumbersOnly</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Poids</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lePoids">
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly|Qt::ImhFormattedNumbersOnly</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Prix</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lePrix">
           <property name="inputMethodHints">
            <set>Qt::ImhFormattedNumbersOnly</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbCommander">
           <property name="text">
            <string>Commander</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="add_pushButton">
           <property name="text">
            <string>Ajouter équipement</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
